src/variables.scss

```diff
@@ -1 +1,5 @@
 $default-red: #cc0028;
+$first-color: black;
+$second-color: white;
```

Comment:
I think it might be unnecessary to have variables for black and white. Its faster to just write black or white. But at the same time its easy to change in the future.

Reply:
That what I was thinking. If we decide ever :) to change anything it is just to change variables.